Code/Python/Transpose_of_matrix.py

```diff
@@ -0,0 +1,62 @@
+'''
+PROBLEM STATEMENT:
+Given a matrix with m rows and n columns. Print its transpose. Transpose of a matrix is obtained by 
+changing the rows to columns and columns to rows i.e., by changing mat[i][j] to mat[j][i].
+'''
+
+
+#A function to print the transpose of the given matrix
+def Transpose(mat, m, n):
+	#Creating a matrix with n rows and m columns
+	trans = [[0 for y in range(m)] for x in range(n)]
+	for x in range(n):
+		for y in range(m):
+			trans[x][y] = mat[y][x]
+	#Printing the transposed matrix
+	print(trans)
+
+
+# Driver Code
+
+# User inputs the the dimension of matrix 
+m = int(input("Enter the no. of rows: "))
+n = int(input("Enter the no. of columns: ")) 
+
+# Creating a matrix with m rows and n columns
+mat = [[0 for j in range(n)] for i in range(m)]
+print("Enter elements of matrix:")
+# User inputs matrix elemnents
+for i in range(m):
+	for j in range(n):
+		num = int(input())
+		mat[i][j] = num
+		
+# Printing the entered matrix
+print("The entered matrix is:\n", mat)
+# A function call to transpose the matrix
+print("The transpose of the entered matrix is:") 
+Transpose(mat, m, n)
+
+
+'''
+TEST CASES:
+1.
+Input:
+m = 4
+n = 3
+[[1, 2, 2], [2, 3, 3], [3, 4, 4], [4, 1, 1]]

```

Comment:
4
3
1
2
2
2
3
3
3
4
4
4
1
1
this is the input

Write here exactly the way it appears on yr console and copy paste Input and Output from there